apps/studio/package.json

```diff
@@ -70,6 +70,7 @@
         "use-resize-observer": "^9.1.0"
     },
     "devDependencies": {
+        "@onlook/models": "*",
```

Comment:
This seems to be a different pattern than how the UI package is used?

Reply:
Nvm its devDep because it's not runtime. But I wonder how that's going to play out with Zod since it'll be passed as an object but also generates types

Reply:
fixed